SPECS/libpwquality/libpwquality.spec

```
@@ -58,34 +53,33 @@ pronounceable passwords from Python applications.
 %setup -q
 
 %build
-[ -f /usr/bin/python3 ] || ln -s /usr/bin/python3 /bin/python
+[ -f %{_bindir}/python3 ] || ln -s %{_bindir}/python3 /bin/python
 
 %configure \
 	--with-securedir=%{_moduledir} \
 	--with-pythonsitedir=%{python3_sitearch} \
-	--with-python-binary=%{__python3} \
+	--with-python-binary=python3 \
```

We've recently discovered that linter's automatic change of `%{__python3}` to `python3` may cause run-time issues (I think the issue was with `#!/usr/bin/python3` lines in scripts getting replaced with `#!python3`). Sorry for the trouble, but could you revert this line in that case to be safe?

```suggestion
	--with-python-binary=%{__python3} \
```